Makefile

```diff
@@ -0,0 +1,19 @@
+# This makefile uses only POSIX 'make' features, except for the
+# following widely implemented extensions -- the .PHONY target, which
+# marks targets that don't generate a file whose name is the same as the
+# target, and prerequisites that depend on glob expansion.
+
+.POSIX:
+.SUFFIXES:
+
+.DEFAULT: preview
+.IGNORE: preview
+.PHONY: preview
+preview:
+	swift package --disable-sandbox preview-documentation --target TSPL
+
+.PHONY: archive
+archive: .build/plugins/Swift-DocC/outputs/TSPL.doccarchive
+
+.build/plugins/Swift-DocC/outputs/TSPL.doccarchive: Sources/TSPL/TSPL.docc/*/*.md
+	xcrun swift package plugin generate-documentation --target TSPL --transform-for-static-hosting
```

Comment:
I _think_ we should be able to simplify this to just:

```suggestion
	swift package generate-documentation
```

The Swift Book has a single target so the plugin will infer the `--target TSPL` piece. Additionally `plugin` is no longer required, you should be able to invoke `swift package generate-documentation` directly.

Reply:
Changed in 5a8889e2.